src/app/components/dialog/dialog.ts

```diff
@@ -205,7 +205,7 @@ export class Dialog implements AfterViewInit,AfterViewChecked,OnDestroy {
                 setTimeout(() => {
                     let height = this.domHandler.getOuterHeight(this.containerViewChild.nativeElement);
 
-                    if(height !== this.currentHeight) {
+                    if((height !== this.currentHeight) && (Math.abs(this.currentHeight - height) != 1)) {
```

Comment:
@jimccfun since we are modifying a component from the framework, will this affect the view anywhere else?
Is it possible to make the modification outside of the component base code? Maybe using an external stylesheet ?

Reply:
After discussing offline I realise that this is an issue across the application and we need to modify the component.